src/views/Character/Character.test.jsx

```diff
@@ -0,0 +1,24 @@
+import { render, screen } from '@testing-library/react';
+import Character from './Character';
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+import { mockSingleCharacters } from '../../__mock__/testdata';
+
+const server = setupServer(
+  rest.get(`https://api.disneyapi.dev/characters/`, (req, res, ctx) => {
```

Comment:
This is the route for all characters. There is most likely a yellow warning in your test telling you that a request is not being intercepted.  MSW is therefore not having any effect on this test